INSTRUCTION.md

```diff
@@ -0,0 +1,24 @@
+# How to validate the changes
+
+#Use this command to run Bootstrap Script
+
+```
+./bootstrap.sh
+```
+
+Use this commands to verify Deployment
```

Comment:
The phrase 'Use this commands' should be corrected to 'Use these commands' for grammatical accuracy.